open-source-policy/practice.md

```diff
@@ -39,43 +39,41 @@ government or non-government -- before writing custom tools. When existing
 libraries need to be modified or improved, CISA staff should make the
 modifications with eventual upstream contribution in mind.
 
-In practice, this generally involves forking the relevant repository to the CISA
-organization within GitHub, creating a new branch with the modifications, and
-sending a pull request to upstream from the CISA fork. Unlike our own projects,
-there is no need for internal code review in this scenario (though it doesn't
-hurt).
+In practice, this generally involves forking the relevant repository to the
+CISA organization within GitHub (`cisagov`), creating a new branch with the
+modifications, and sending a pull request to upstream from the CISA fork.
+Unlike our own projects, there is no need for internal code review in this
+scenario (though it doesn't hurt).
 
 In terms of licensing: as works of the government, employee contributions are
 public domain in the United States, regardless of the outside project's
 contribution agreement. This does not change the overall license status of the
 outside project.
 
-As [the Free Software Foundation
-says](https://www.gnu.org/licenses/gpl-faq.html#GPLUSGovAdd) about
-government-contributed improvements to GPL software:
+As [the Free Software Foundation says](https://www.gnu.org/licenses/gpl-faq.html#GPLUSGovAdd)
+about government-contributed improvements to GPL software:
 
-> Yes. If the improvements are written by US government employees in the course
-> of their employment, then the improvements are in the public domain. However,
-> the improved version, as a whole, is still covered by the GNU GPL. There is no
-> problem in this situation.
+> Yes. If the improvements are written by US government employees in the
+course of their employment, then the improvements are in the public domain.
+However, the improved version, as a whole, is still covered by the GNU GPL.
+There is no problem in this situation.
 
-See also: [The Department of Defense's FAQ question about
-this](http://dodcio.defense.gov/Open-Source-Software-FAQ/#Q:_Can_government_employees_contribute_code_to_open_source_software_projects.3F).
+See also: [The Department of Defense's FAQ question about this](http://dodcio.defense.gov/Open-Source-Software-FAQ/#Q:_Can_government_employees_contribute_code_to_open_source_software_projects.3F).
 
 ### Contributor License Agreements (CLAs) ###
 
 Some external projects have CLAs. You cannot sign these yourself, in your
 official capacity.
 
-1. See if there is an organizational CLA available.
-1. Send the agreement to DHS's Office of General Counsel (OGC) for review.
-    * Email cisagov-github-group@trio.dhs.gov for who the best contact is.
+1. See if there is an organizational CLA available
+1. Send the agreement to DHS's Office of General Counsel (OGC) for review
+    * Email cisagov-github-group@trio.dhs.gov for who the best contact is
```

Comment:
I think "for who the best contact is" :arrow_right: "to determine the best contact" is less awkward.

Reply:
I like it! Updated in 267f33866baf7df7d71137c503c07968d32d75e2

Reply:
What I meant was
```
    * Email cisagov-github-group@trio.dhs.gov to determine the best contact.
```

Reply:
Changed in commit ad35141.